tests/conditional_compilation/test_verify.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+""" Test to verify inference results.
+"""
+
+import sys
+import glob
+from inspect import getsourcefile
+from pathlib import Path
+import numpy as np
+from proc_utils import cmd_exec  # pylint: disable=import-error
+from install_pkg import get_openvino_environment  # pylint: disable=import-error
+
+
+def run_infer(model, out, install_dir):
+    """ Function running inference
+    """
+
+    cmd_exec(
+        [sys.executable,
+         str((Path(getsourcefile(lambda: 0)) / ".." / "tools" / "infer_tool.py").resolve()),
+         "-d=CPU", f"-m={model}", f"-r={out}"
+         ],
+        env=get_openvino_environment(install_dir),
+    )
+
+
+def test_verify(test_id, model, artifacts, openvino_root_dir, tolerance=0.000001):  # pylint: disable=too-many-arguments
+    """ Test verifying that inference results are equal
+    """
+    out = artifacts / test_id
+    install_prefix = artifacts / test_id / "install_pkg"
+    run_infer(model, out, openvino_root_dir)
```

Comment:
`infer_tool.py` via `-r` consumes full path to an .npz. So, instead of `out`, there should be an `out / infer.npz` or smth like that. And, when path to .npz is specified, `assert len(results) == 2` is not required at all